Question2.cs

```diff
@@ -14,20 +14,16 @@ public class Question2
     {
         string connectionString = "Data Source=(local);Initial Catalog=Northwind;Integrated Security=true";
 
-        // Provide the query string with a parameter placeholder.
-        string queryString = "SELECT ItemId, FirstName, LastName "
-                + "FROM People "
-                + "WHERE Age > @age "
-                + "ORDER BY UnitPrice DESC;";
-
-        // Specify the parameter value.
-        int paramValue = 25;
-
-        public DataTable ReadPersonData()
+        public DataTable ReadPersonData(string jobTitle)
         {
             SqlConnection connection = new SqlConnection(connectionString);
-            var command = new SqlCommand(queryString, connection);
-            command.Parameters.AddWithValue("@age", paramValue);
+            
+            var queryString = "SELECT ItemId, FirstName, LastName "
+                + "FROM People "
+                + $"WHERE JobTitle = '{jobTitle}'"
+                + "ORDER BY Age DESC;";
+
+            var command = new SqlCommand(queryString, connection);            
```

Comment:
var command = new SqlCommand($"SELECT ItemId, FirstName, LastName FROM People WHERE JobTitle = '{jobTitle}' ORDER BY Age DESC;", connection);

Reply:
Not sure if this was intentional, but there will be no space between job title and the Order by statement. :) which can be considered a check for being attentive.

Reply:
Certainly not deliberate :)